templates/_devcontainer/parcel/run.sh.ts

```diff
@@ -14,7 +14,7 @@ npm install
 
 npm run watch:parcel
 `;
-	return template.trim();
+	return template.trim().replace(/\r\n/g, '\n');
```

Comment:
You probably need to set `templateFunction.noReformat = true` or `formatFiles` will replace that with `os.EOL`.